File: app/src/main/java/com/example/seki/training_retrofit/client/WeatherService.java
package com.example.seki.training_retrofit.client;

import com.example.seki.training_retrofit.WeatherData;

import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Query;

public interface WeatherService {
    @GET("json/v1")
    Call<WeatherData> singleWeather(@Query("city") String city);
}

File: app/src/main/java/com/example/seki/training_retrofit/client/WeatherService.kt
package com.example.seki.training_retrofit.client;

import com.example.seki.training_retrofit.WeatherData

import retrofit2.Call
import retrofit2.http.GET
import retrofit2.http.Query

interface WeatherService {
    @GET("json/v1")
    fun singleWeather(@Query("city") city: String): Call<WeatherData>
}